### DISCUSSION TITLE
Case for Inter but Multiplexed/Uniwidth?

### DISCUSSION THREAD

**Discussion by pietrocasella** [rsms/inter#637] [Ideas]
TLDR: Inter should have a variant that is Multiplexed/Uniwidth similar to https://maxitype.com/typeface/selecta/. Does not have to be perfect, meaning we can probably simply play with spacings for a first version, but should not "break lines" when doing weight changes
![image](https://github.com/rsms/inter/assets/5515131/ada70977-e90d-44ba-86ce-639efd76f8b8)

Motivation: 
With the increased importance of text based user experiences such as Chatbots, using weight-based animations will be an important affordance. We've all seen "cursor" affordance with ChatGPT, I believe similar things will happen with other text animations. I also think we will see font symbols as carriers of further affordances for this same end. 

For example when a user hovers a word and we want that word to become "bold" (simple case) without breaking the text. A more interesting case is when an AI wants to enrich the text with metadata, a subtle way to do is a font-weight animation "pulsing" effect. The Selecta website has plenty of examples of how this animation looks like or simply googling for uniwdith fonts.

Is there an easy way to do this for a v1? Can you help me navigate the process of trying something like this?
Cheers. P.
